pkg/csi/service/node.go

```diff
@@ -1069,6 +1092,32 @@ func getDevice(path string) (*Device, error) {
 	}, nil
 }
 
+func rescanDevice(ctx context.Context, dev *Device) error {
+	ctx = logger.NewContextWithLogger(ctx)
+	log := logger.GetLogger(ctx)
+
+	devRescanPath, err := getDeviceRescanPath(dev)
+	if err != nil {
+		return err
+	}
+
+	err = ioutil.WriteFile(devRescanPath, []byte{'1'}, 0666)
+	if err != nil {
+		mssg := fmt.Sprintf("Error rescanning block device %q. %v", dev.RealDev, err)
+		log.Error(mssg)
+		return fmt.Errorf(mssg)
+	}
+	return nil
+}
+
+func getDeviceRescanPath(dev *Device) (string, error) {
+	parts := strings.Split(dev.RealDev, "/")
```

Comment:
Can you provide an example ReadDev here and how we are splitting it with "/" is helping us? Its hard to know what parts[2] really is.

Reply:
Exmaple `RealDev` is `/dev/sda`

Reply:
Done